SUBMISSION_PROCESS_RULES.md

```diff
@@ -0,0 +1,176 @@
+
+The sampling code for the held-out workloads and the hyperparameters is publicly available (**TODO link to both functions!**). Both sampling functions take as input a random seed, which will be provided by a trusted third party after the submission deadline.
+
+### Leaderboard
+
+The publication of the results will contain two separate leaderboards, one for the self-tuning and one for the external tuning ruleset. All valid submissions will be ranked by the benchmark score, taking into account all workloads, including the held-out ones. The leaderboard will clearly mark scores that were verified by the working group.
+
+## Sprit jury & challenging submissions
+
+The spirit jury will be responsible for deciding whether a submission violates the "spirit of the rules". Submitters may challenge other submissions, i.e. request a review by the spirit jury to determine whether a submission violates the rules of the competition. To challenge a submission, please write an email to <algorithms-chairs@mlcommons.org> with the subject "[Challenge] *submission_name*". The email needs to link to the challenged submission and include a detailed description of why the submission should be reviewed.
+
+The spirit jury may then hear the justifications of the submitters, inspect the code, and also ask the submitters to explain how the submission was produced, for example, by disclosing their intermediate experiments. Example cases that might be reviewed by the spirit jury are cases of multiple similar submissions by the same submitter or extensive workload-specific tuning.
+
+**TODO Who is on the Jury?**
+
+## Awards and prize money
+
+An awards committee will award a prize for the "*Best Performance*" in each ruleset as well as a "*Jury Award*". The prize for the best-performing submission will take into account the [benchmark score](RULES.md#benchmark-score-using-performance-profiles) on the full benchmark. The "*Jury Award*" will favor more out-of-the-box ideas that show great potential, even though the method may not be of practical value with the current landscape of models, software, etc.
+
+The prize money for "*Best Performance*" in a ruleset is $20,000 each. The winner of the "*Jury Award*" will be awarded $10,000. We reserve the right to split the prize money and distribute it among multiple submissions.
+
+### Awards committee
+
+The awards committee will be responsible for awarding prize money to submissions. Members of the awards committee can suggest submissions to be considered for the awards. The committee will vote on the winning submissions, the submission with the most votes in each respective category wins the awards, and if eligible, the prize money.
```

Comment:
Maybe we should let the committee itself decide how best to operate and make this a bit more vague. "The committee will try to reach a consensus on consensus on how to award prize money and settle disagreements by majority vote, if necessary."

And maybe we should mention that a submission can be a "winning" submission without being eligible for prize money? Or how should that work? Should the committee identify both the winning submission and the best prize money eligible submission per category? I think we might want to clarify this in a working group meeting as well.

Reply:
I changed the section now to:

The awards committee will be responsible for awarding prize money to submissions. The committee will try to reach a consensus on how to award prize money and settle disagreements by majority vote, if necessary.

Reply:
The Awards and prize money section has an additional paragraph stating:

If a submission is ineligible to win prize money it can still win an award. The prize money will then go to the highest-ranking eligible submission.

Reply:
I think if @georgedahl submits an algorithm and it is the best-performing one, it should indeed be winning. And I think it is fair if the prize money then goes to the best-performing submission that is eligible for prize money.

I don't know whether we need to specify or discuss how we handle baselines. What if one of those wins?